.github/workflows/test.yml

```diff
@@ -23,3 +23,17 @@ jobs:
     
       - name: Execute Gradle test
         run: ./gradlew test
+
+      - name: Upload test results
+        uses: actions/upload-artifact@v2
+        if: success() || failure()
+        with:
+          name: test-results
+          path: build/test-results/**/*.xml
```

Comment:
Can we use 
```
path: |
    build/test-results/**/*.xml
    build/reports/
```

and it has v3 now?

Reply:
@kojilin 
I think we can use v3 . Let me try 🙆‍♂️ 

However, separating the upload may be easier for `test-report.yml` to [reuse the artifacts](https://github.com/line/thrift-gradle-plugin/pull/7/files#diff-f23e09a20042156596d85e139423b1991e5ad593ee212cbdacecdef8edcc3027R14) for creating test report?